tendrl/gluster_integration/manager/manager.py

```diff
@@ -200,6 +266,27 @@ def on_pull(self, raw_data, cluster_id):
                     index += 1
                 except KeyError:
                     break
```

Comment:
Make sense, but question here should we really need to break out if there is a KeyError raised while processing one of the volumes. I think we should not rather we should proceed further with other volumes. Does my understanding correct. Also we need to log the details as well here